test/helpers/git-utils.js

```diff
@@ -212,6 +209,15 @@ export async function gitCommitTag(gitHead) {
  * @param {String} branch The branch to push.
  * @throws {Error} if the push failed.
  */
-export async function push(origin, branch) {
+export async function push(origin = 'origin', branch = 'master') {
```

Comment:
can you please explain why you set defaults now? Could this be  problem if `master` is not the default branch on remote?

And unrelated to this PR: should the `origin` argument be called `remote`? `"origin"` is the default remote name, or does the parameter refer to something else here?

Reply:
This code is not used in semantic-release. It's just code used in the test.
I set the defaults out of convenience.

Yes I can change the arg name to `remote`

Reply:
Well actually in each other function on this util file similar args are named `origin`. Should I change it to `remote` everywhere? Or maybe `repositoryUrl` would be clearer?

Reply:
`repositoryUrl` is more clear I think 👍